frigate/api/defs/review_query_parameters.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime, timedelta
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class ReviewQueryParams(BaseModel):
+    cameras: Optional[str] = "all"
+    labels: Optional[str] = "all"
+    zones: Optional[str] = "all"
+    reviewed: Optional[int] = 0
+    limit: Optional[int] = None
+    severity: Optional[int] = None
```

Comment:
This is wrong. It should be a string. Example request: `http://localhost:5173/api/review?limit=10&severity=alert`

Reply:
Fixed
<img width="519" alt="image" src="https://github.com/user-attachments/assets/c268f1a1-b825-4cc7-bb0d-53c9038c1584">